src/App.css

```diff
@@ -2,6 +2,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  margin-bottom: 50px;
```

Comment:
# Your Design 
![image](https://user-images.githubusercontent.com/30873007/209093427-cc91be44-3122-482c-9b55-dc4f01af54a6.png)


# Original Design
![image](https://user-images.githubusercontent.com/30873007/209093548-e4c54853-f89b-4ad4-8fb4-bc3f59d8bb63.png)

Please decrease the `margin-bottom: 50px` in accordance to the match the original design.